app/src/main/AndroidManifest.xml

```diff
@@ -5,9 +5,23 @@
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
-        android:supportsRtl="true"
         android:theme="@style/AppTheme">
 
+        <activity android:name="lt.vilnius.tvarkau.StartActivity" />
+
+        <!-- This strangely named alias is here for backwards compatibility.
+             Renaming activity with LAUNCHER category means that shortcuts created by users
+             on their phones with previous version of the app will disappear or stop working. -->
+        <activity-alias
+            android:name="apps.factory.emiestas.MainActivity"
```

Comment:
Then, please reopen https://github.com/vilnius/tvarkau-vilniu/issues/2 and let's discuss that with your pros/cons for that.
I would rather manage 2 apps instead of 1, than have that ugly package id. Then we force those few users to update to new app and unpublish this old app too.

If we don't change this legacy package id now, then we won't change it in the future for sure.